src/theme/index.ts

```diff
@@ -33,10 +35,50 @@ export const theme = extendTheme(vtheme, {
       ':root': {
         '--box-shadow': '0px 2px 4px lightgray',
         '--box-shadow-darker': '0px 2px 4px #808080b5',
+        '--read-more-from-light': 'rgba(245, 245, 247, 0)',
+        '--read-more-to-light': '#F5F5F7',
+        '--read-more-from-dark': 'rgba(26, 32, 44, 0)',
+        '--read-more-to-dark': '#1A202C',
```

Comment:
There's no need to define custom variables this way. You can use the `colors.ts` to define the colors there, and then you can access them via `var(--chakra-colors-name-of-your-color)`.